kb/temporal-platform-limits-sheet.md

```diff
@@ -9,8 +9,11 @@ date: 2022-06-07T00:00:00Z
 ---
 
 Running into limits can cause unexpected failures, so be mindful when you design your systems.
+
 Here is a list of many hard (error) or soft (warn) limits that you could encounter while using the Temporal Platform.
 
+Unless otherwise stated, Event limits apply to all Events running in parallel on a system.
```

Comment:
For Events, "occurring" seems to be a better verb than "running". I'm also unsure what "in parallel" means in this context. I doubt that "simultaneously" is accurate, but I don't know what bounding period might be appropriate.